integrate-kobiton-travisci.md

```diff
@@ -0,0 +1,190 @@
+# Run Kobiton Automation Test on Travis CI
+This document will guide you to prepare an app automation test project on GitHub and use Travis CI to automatically execute the test on Kobiton devices everytime you push a commit to GitHub.
+
+## Table of contents 
+  - [Prerequisites](#prerequisites)
+  - [1. Configure Travis CI with GitHub repository](#1-configure-travis-ci-with-github-repository)
+  - [2. Configure automation test script](#2-configure-automation-test-script)
+  - [3. Run automation script on Kobiton devices](#3-run-automation-script-on-kobiton-devices)
+  - [4. Fetch session data through REST API](#4-fetch-session-data-through-REST-API)
+  - [5. (Optional) Travis CI customization](#5-optional-travis-ci-customization)
+  - [6. Feedback](#6-feedback)
+
+## Prerequisites
+  - Kobiton account
```

Comment:
`A Kobiton account`. And how about Travis account and Hockey App account?

Reply:
@thangvo38 Do you have any changes or suggestion ?

Reply:
Travis CI uses github account to login. The instruction on how to sync travis -> github is in the next section